src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcScriptsSpec.scala

```diff
@@ -38,11 +38,12 @@ class hmrcScriptsSpec
   "hmrcScripts" should {
     "include the hmrc-frontend script tag" in {
       val hmrcScripts = app.injector.instanceOf[HmrcScripts]
+      hmrcfrontend.RoutesPrefix.setPrefix("/foo-service/hmrc-frontend")
       val scripts     = hmrcScripts().select("script")
 
       scripts                   should have size 1
       scripts.first.attr("src") should fullyMatch regex
-        """/assets/hmrc-frontend-\d+.\d+.\d+.min.js""".r
+        """/foo-service/hmrc-frontend/assets/hmrc-frontend-\d+.\d+.\d+.min.js""".r
```

Comment:
@matthewmascord just to check, is this change intentional or part of something else?

Reply:
There is a race condition around this because the reverse router is statically initialised and it was failing locally. So this forces it to have a certain value.